src/v/kafka/server/request_context.h

```diff
@@ -248,12 +254,71 @@ class request_context {
         return _conn->server().credentials();
     }
 
+    // Set force to true to always enqueue the audit event regardless of event
+    // type
```

Comment:
chore: update outdated comment

Reply:
For those wondering - the purpose of `additional_resources` is to the audit event contains all relevant information about the action.  Each `authz` check result is stored in request_context and if the request is _per_ resource, then we have that data, but not all authz checks are per resource, some just do a general "cluster level" authz check (e.g. create-topics).  In those situations, we may only have a single authz call and no relevant data.  The purpose of `additional_resources` is to fill in the gaps, if you will, in those situations.